README.md

```diff
@@ -18,9 +18,8 @@ Each folder in this repository except parquet has two items within:
 
  - `README` - An explanation of everything about the data
  - `csv` - A folder with all relevant data in either CSV or TSV format
-
-parquet instead contains a number of standalone .parquet files with different data formats
-
+ 
+parquet instead contains a `parquet` folder with all relevant data in a a variety of parquet formats
```

Comment:
This is exactly what should *NOT* be happening.  `parquet` is a dumping ground type of organization that will make this repo useless in the long run -- code rot.  Every other folder is for a specific dataset.  If this dataset is taxi, call it taxi.  If we have both csv and parquet data, put them both there.  The reame should reflect the dataset.  We may want to use these files for many examples in the future, not just for showing a parquet load.  In that case, the readme should explain the data set so that anyone who uses it understand what is in the table(s).